kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/AdminResourceSuite.scala

```diff
@@ -125,6 +125,19 @@ class AdminResourceSuite extends KyuubiFunSuite with RestFrontendTestHelper {
     assert(200 == response.getStatus)
   }
 
+  test("refresh limited users of the kyuubi server") {
+    var response = webTarget.path("api/v1/admin/refresh/limited_users")
+      .request()
+      .post(null)
+    assert(405 == response.getStatus)
```

Comment:
`403 Forbidden` may be a more suitable response code than `405 Method not allowed`.

Reply:
I wrote the code based on other test cases. If you think the 403 status code should be better, I can modify it (including all other test cases).